LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 - Sygic a.s.
```

Comment:
asi opytam sa Feriho, ako to je s licenciou, pod ktorou by sme to cheli zverejnovat. Dufam, ze ostane MIT

Reply:
Oki, použil jsem tu stejnou, co dali kluci z iOS ;)